Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -353,6 +356,303 @@ void MolDraw2D::drawMolecule(const ROMol &mol, const std::string &legend,
   }
 }
 
+namespace {
+void get2DCoordsMol(RWMol &mol, double &offset, double spacing, double &maxY,
+                    double &minY, int confId, bool shiftAgents,
+                    double coordScale) {
+  try {
+    MolOps::sanitizeMol(mol);
+  } catch (const MolSanitizeException &e) {
+    mol.updatePropertyCache(false);
+    try {
+      MolOps::Kekulize(mol, false);  // kekulize, but keep the aromatic flags!
+    } catch (const const MolSanitizeException &e) {
```

Comment:
gadzooks... that was a stupid one.
Amusing that it still built on the mac.

Anyway, it's fixed.